app/javascript/react/screens/App/Overview/Overview.js

```diff
@@ -334,12 +334,15 @@ class Overview extends React.Component {
     // Full-height grey background (.cards-pf) for in-progress cards and empty states, otherwise only grey behind aggregate cards
     const overviewContent =
       inProgressCardsVisible || emptyStateVisible ? (
-        <div className="row cards-pf" style={{ overflow: 'auto', paddingBottom: 50, height: '100%' }}>
+        <div
+          className="row cards-pf"
+          style={{ overflow: 'auto', overflowX: 'hidden', paddingBottom: 50, height: '100%' }}
```

Comment:
This will solve the problem by preventing `overflow-x: auto` rather than having to override it:

```suggestion
          style={{ overflowY: 'auto', paddingBottom: 50, height: '100%' }}
```